src/Components/Location.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react'
+import "./locations.json"
+
+
+const Location = () => {
+  const Locate_ApI = require("./locations.json")
+  const [data, setData] = useState([])
+  
+
+  useEffect(() => {
+    setData(Locate_ApI);
```

Comment:
Technically you could just do this instead of making a fetch request since they're both local, so not gonna take points off for this